plugin.rb

```diff
@@ -104,6 +104,7 @@ module ::DiscourseAutomation
           automation.trigger!(
             "kind" => DiscourseAutomation::Triggerable::USER_ADDED_TO_GROUP,
             "usernames" => [user.username],
+            "user" => user,
```

Comment:
I regret having done this "usernames" things here in the past. Given it's user_added_to_group, it can only be one user anyways. I don't want to break backwards compat but from now I will try to use `user` instead. And might deprecate/remove old usage where easy later.